clusters/app.ci/assets/admin_openshift-private-ci-to-qe-private-deck-ns_rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: openshift-private-ci-to-qe-private-deck-ns
+  namespace: qe-private-deck
+subjects:
+- kind: Group
+  apiGroup: rbac.authorization.k8s.io
+  name: openshift-private-ci
+roleRef:
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+  name: admin
```

Comment:
@smg247 For rover group openshift-private-ci, we want to grant it **view** permission. May i know which CluterRole should I use here ?

Reply:
Let's try `cluster-reader`

Reply:
Update the role as suggested.